### PR TITLE
Endpoint for pending invites

### PR DESCRIPTION
**Pull request by gevorgmansuryan**
https://github.com/humhub/rest/issues/161

### CODE REVIEW

File: models/Invite.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+use humhub\modules\space\models\Space;
+use humhub\modules\user\models\User;
+
+/**
+ * @property User|null $originator
+ * @property User|null $createdBy
+ * @property User|null $updatedBy
+ * @property Space|null $space
+ */
+class Invite extends \humhub\modules\user\models\Invite
```
Comment (luke-) [inline models/Invite.php:14]: Can you please create a PR for the Core `develop` branch here.
And mark the file here that it is required up to version 1.17?

**Review by luke- — CHANGES_REQUESTED** [review]
@gevorgmansuryan Thanks looks good. See one point below.

- Can you also add this to the CHANGELOG and regenerate the docs?

File: models/Invite.php (hunk 1 above)
Comment (gevorgmansuryan) [reply models/Invite.php:14]: @luke- though we needed it asap, that's why did it here instead of core. Ok, got it

Comment (luke-) [reply models/Invite.php:14]: @gevorgmansuryan Your temporary solution is fine. But let's also add this in core.

File: module.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "version": "0.10.0",
     "homepage": "https://github.com/humhub/rest",
     "humhub": {
-        "minVersion": "1.16"
+        "minVersion": "1.16.2"
```
Comment (luke-) [inline module.json:11]: `v1.16`?

**Review by luke- — CHANGES_REQUESTED** [review]
We can undo that, right?

**Pull request merged**